docs/development/How to build AL distribution.md

```diff
@@ -0,0 +1,60 @@
+---
+title: 'AL Building'
+---
+
+# How to build AlmaLinux OS distribution on one's own
+
+## Hardware and software requirements to a final machine 
+
+* AlmaLinux 9
+* 6+ CPU cores
+* 8+ GB RAM
+* 300+ GB free storage space
+
+## Deploying a Pungi node
+
+That Ansible scenario prepare your machine to building a distribution
```

Comment:
Maybe, something like:
"""
The following process will guide you through the process of:
1. Installing Docker CE and prepare docker containers to build the distributions inside them
2. Setting up a __systemd timer__ to synchronize a local mirror of packages, which will be used later for building the distribution
"""